### PR TITLE
Bump xpaas to 1.4.8

### PR DESCRIPTION
**Pull request by rcernich**
These changes update the xpaas templates to the latest release and update the list of templates made available on OSO.

### CODE REVIEW

File: official.yaml [hunk 1]
```diff
@@ -442,21 +485,30 @@ data:
         docs: https://github.com/jboss-openshift/application-templates/blob/{xpaas_version}/docs/webserver/jws31-tomcat7-postgresql-s2i.adoc
       - location: https://raw.githubusercontent.com/jboss-openshift/application-templates/{xpaas_version}/webserver/jws31-tomcat8-basic-s2i.json
         docs: https://github.com/jboss-openshift/application-templates/blob/{xpaas_version}/docs/webserver/jws31-tomcat8-basic-s2i.adoc
+      - location: https://raw.githubusercontent.com/jboss-openshift/application-templates/{xpaas_version}/webserver/jws31-tomcat8-https-s2i.json
+        docs: https://github.com/jboss-openshift/application-templates/blob/{xpaas_version}/docs/webserver/jws31-tomcat8-https-s2i.adoc
```
Comment (bparees) [inline official.yaml]: this change moved the online-start/pro tags from the "basic" template to this template.  intentional?

Comment (rcernich) [reply official.yaml]: Yes.  The availability changes were directed by the product managers.  @rettori can confirm.

**Comment by bparees** [comment]
ok, lgtm but i'm not the right person to sanity check the actual tag updates... is @rettori the right person to do that?

**Comment by rcernich** [comment]
Probably.  I've also asked the product teams to review it.  That said, there may be additional PRs, as I don't think all of the PMs provided input to @rettori.

**Pull request merged**